content/en/docs/concepts/services-networking/service.md

```diff
@@ -827,434 +628,268 @@ metadata:
 
-        # Specifies the bandwidth value (value range: [1,2000] Mbps).
-        service.kubernetes.io/qcloud-loadbalancer-internet-max-bandwidth-out: "10"
+Kubernetes also supports DNS SRV (Service) records for named ports.  If the
+`my-service.my-ns` Service has a port named `http` with the protocol set to
+`TCP`, you can do a DNS SRV query for `_http._tcp.my-service.my-ns` to discover
+the port number for `http`, as well as the IP address.
 
-        # When this annotation is set，the loadbalancers will only register nodes
-        # with pod running on it, otherwise all nodes will be registered.
-        service.kubernetes.io/local-svc-only-bind-node-with-pod: true
-```
+The Kubernetes DNS server is the only way to access `ExternalName` Services.
+You can find more information about `ExternalName` resolution in
+[DNS Pods and Services](/docs/concepts/services-networking/dns-pod-service/).
 
-### Type ExternalName {#externalname}
 
-Services of type ExternalName map a Service to a DNS name, not to a typical selector such as
-`my-service` or `cassandra`. You specify these Services with the `spec.externalName` parameter.
+## Traffic policies
 
-This Service definition, for example, maps
-the `my-service` Service in the `prod` namespace to `my.database.example.com`:
+You can set the `.spec.internalTrafficPolicy` and `.spec.externalTrafficPolicy` fields
+to control how Kubernetes routes traffic to healthy (“ready”) backends.
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: my-service
-  namespace: prod
-spec:
-  type: ExternalName
-  externalName: my.database.example.com
-```
+See [Traffic Policies](/docs/reference/networking/virtual-ips/#traffic-policies) for more details.
 
-{{< note >}}
-ExternalName accepts an IPv4 address string, but as a DNS name comprised of digits, not as an IP address.
-ExternalNames that resemble IPv4 addresses are not resolved by CoreDNS or ingress-nginx because ExternalName
-is intended to specify a canonical DNS name. To hardcode an IP address, consider using
-[headless Services](#headless-services).
-{{< /note >}}
+## Custom IP address configuration for `type: NodePort` Services {#service-nodeport-custom-listen-address}
 
-When looking up the host `my-service.prod.svc.cluster.local`, the cluster DNS Service
-returns a `CNAME` record with the value `my.database.example.com`. Accessing
-`my-service` works in the same way as other Services but with the crucial
-difference that redirection happens at the DNS level rather than via proxying or
-forwarding. Should you later decide to move your database into your cluster, you
-can start its Pods, add appropriate selectors or endpoints, and change the
-Service's `type`.
+You can set up nodes in your cluster to use a particular IP address for serving
+node port services. You might want to do this if each node is connected to multiple networks
+(for example: one network for application traffic, and another network for traffic between
+nodes and the control plane).
```

Comment:
yeah, we should also encourage to use it, otherwise any port on any IP on that node can be used to reach the service, that have some interesting scenarios with some cloud providers https://github.com/kubernetes/kubernetes/issues/111863

/cc @thockin @danwinship